### ISSUE TITLE
Blitz modifies logging output by rewriting paths, even in error messages

### ISSUE THREAD

**Issue by vincentrolfs** [blitz-js/blitz#2885]
### What is the problem?

```js
  console.log(process.cwd())
  // Output: "/app/.blitz/build"
  console.log(process.cwd() + "/example.txt")
  // Output: "/app/example.txt"
```

It looks like blitz tries to rewrite paths in console output? This also applies to error messages. This is extremely confusing, as I got an error message like

`[Error: ENOENT: no such file or directory, open '/app/foo/bar.pdf']`

when the real error should have been

`[Error: ENOENT: no such file or directory, open '/app/.blitz/build/foo/bar.pdf']`

This cost me hours of debugging, as the directory existed in `/app` but not in `/app/.blitz/build`. The wrong error message was extremely misleading. Please don't modify my error messages.

### Paste all your error logs here:

/

### Paste all relevant code snippets here:

/

### What are detailed steps to reproduce this?

Run the following code in your blitz app:

```js
  console.log(process.cwd())
  // Output: "/app/.blitz/build"
  console.log(process.cwd() + "/example.txt")
  // Output: "/app/example.txt"
```

### Run `blitz -v` and paste the output here:

```
Linux 5.11 | linux-x64 | Node: v14.17.4

blitz: 0.38.6 (global)
blitz: 0.38.2 (local)

  Package manager: yarn 
  System:
    OS: Linux 5.11 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (4) x64 Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
    Memory: 1.36 GB / 15.31 GB
    Shell: 3.3.1 - /usr/bin/fish
  Binaries:
    Node: 14.17.4 - ~/.nvm/versions/node/v14.17.4/bin/node
    Yarn: 1.22.11 - ~/.nvm/versions/node/v14.17.4/bin/yarn
    npm: 6.14.14 - ~/.nvm/versions/node/v14.17.4/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/client: ~2.20 => 2.20.1 
    blitz: 0.38.2 => 0.38.2 
    prisma: ~2.20 => 2.20.1 
    react: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71 
    react-dom: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71 
    typescript: ~4.3.4 => 4.3.4 
```

**Comment by flybayer** [comment]
Hey @vincentrolfs, I'm very sorry for causing you to waste hours on that 😞 

The good news is that this is totally fixed as of v0.40.0! The intermediate `.blitz` folder is no longer used. It's a much better dev and build experience now

**Comment by vincentrolfs** [comment]
@flybayer Thanks a lot for your reply, I will check out the new version!
